packages/typescript/server/README.md

```diff
@@ -124,6 +126,50 @@ const connectResponse = await jamsocket.connect({
 })
 ```
 
+### `status(backendId)`
+
+The Jamsocket instance includes a `status` function that you can use to get the current status of a backend. It takes a backend ID and returns a `Promise<BackendState>`.
+
+If the underlying status request fails with a non-200 code, the returned Promise will `reject` with an `HTTPError`. This includes if no backend is found with the given ID.
+
+Learn more about the various statuses backends can have in [our API docs](https://docs.jamsocket.com/platform/reference/v2#get-a-backends-current-status).
+
+```ts {8}
+import { Jamsocket } from '@jamsocket/server'
+const jamsocket = new Jamsocket({
+  account: '[YOUR ACCOUNT]',
+  token: '[YOUR TOKEN]',
+  service: '[YOUR SERVICE]',
+})
+
+const backendState = await jamsocket.status(backendId)
+```
+
+### `statusStream(backendId, onStatusCallback)`
+
+The Jamsocket instance includes a `statusStream` function that you can use to stream a backend's status updates. It takes a backend ID and an onStatusCallback (`(backendState: BackendState) => void`), which will be called on each status as it is received. The `statusStream()` function returns a `Promise<UnsubscribeFn>`. The underlying stream is closed when either the backend reaches a `terminated` state or the `UnsubscribeFn` is called. If the underlying stream closes but the backend has not terminated and the `UnsubscribeFn` has not been called, this function will automatically reconnect.
```

Comment:
does this function need to be async? It seems like it could just return the `UnsubscribeFn` and start the async task in the background.

not a blocker on this PR

Reply:
It's kinda subtle, but the reason I'm returning a `Promise` here is because if the response from the initial HTTP request is a 404 (or really any other error) this function will throw an `HTTPError`, and the only way for a dev to catch that is if we return the Promise chain that it was thrown in.